packages/client/tests/functional/findFirstOrThrow/tests.ts

```diff
@@ -83,22 +83,16 @@ testMatrix.setupTestSuite((suiteConfig, suiteMeta) => {
   })
 
   test('does not accept rejectOnNotFound option', async () => {
-    const record = prisma.user.findFirstOrThrow({
-      where: { email: existingEmail },
-      // @ts-expect-error passing not supported option on purpose
-      rejectOnNotFound: false,
-    })
-
-    await expect(record).rejects.toThrowErrorMatchingInlineSnapshot(`
-
-                                    Invalid \`prisma.user.findFirstOrThrow()\` invocation in
-                                    /client/tests/functional/findFirstOrThrow/tests.ts:86:32
-
-                                       83 })
-                                       84 
-                                       85 test('does not accept rejectOnNotFound option', async () => {
-                                    →  86   const record = prisma.user.findFirstOrThrow(
-                                    'rejectOnNotFound' option is not supported
-                              `)
+    expect.assertions(1)
+
+    try {
+      await prisma.user.findFirstOrThrow({
+        where: { email: existingEmail },
+        // @ts-expect-error passing not supported option on purpose
+        rejectOnNotFound: false,
+      })
+    } catch (error) {
+      expect(error.message).toContain(`'rejectOnNotFound' option is not supported`)
+    }
```

Comment:
```suggestion
    const record = prisma.user.findFirstOrThrow({
      where: { email: existingEmail },
      // @ts-expect-error passing not supported option on purpose
      rejectOnNotFound: false,
    })

    await expect(record).rejects.toThrowError(/'rejectOnNotFound' option is not supported/)
```

Reply:
Actually, looking at the docs, I think a regexp is not even needed here, it should match a substring and not the whole message if just a string is passed